smartweave/scripts/test-states/relay-registry-init-state.json

```diff
@@ -171,9 +171,9 @@
     "E616E560EB2F9E380C7DE8506D8EB075517E76DA": "0xa3EE387E97fbeC75A2623A809726c38f6ac19B2D",
     "E6CFF9FD7ACFF8015604931A134AB66A1AE72E8A": "0xf2948eDD428305D28DcBd6C284470e3823ee182b",
     "E7D161A93D2FCFAF53CA35B9AB07F734ECC7D5B5": "0x9DCA83E91041222D20c491F5b25ddC7Ac79Ee53B",
-    "E96BA73676476F68DCCDE99A383E114A33432B04": "0xF8Ae447F48af3748d64C509B20Be78C512292DB1",
-    "EA50D08EC10EE837B8A1719AC4447547712EDB66": "0x7f85F0E167Cc229A7eB9B41221D9D8a7965f2949",
-    "EB0F5258EA351742DB9D79BB8697640658F1D9CF": "0xC2c58DaFE37Ccf2D0bdB0a90194bE969aD05CFFc",
+    "E96BA73676476F68DCCDE99A383E114A33432B04": "0xF8Ae447F48af3748d64C509B20Be78C512292DB1"
+  },
+  "claimable": {
```

Comment:
Should we really change init state?

Reply:
These probably should be local only changes. We used this file to redeploy with transferred state when we migrated contracts early on. Now its not really used so can be re-purposed or removed.